schema/artwork/index.js

```diff
@@ -251,25 +251,16 @@ export const artworkFields = () => {
     },
     is_buy_nowable: {
       type: GraphQLBoolean,
-      description: "When in an auction, can the work be bought before any bids are placed",
+      description: "When in an auction, can the work be bought immediately",
       resolve: ({ id, acquireable, sale_ids }, options, request, { rootValue: { salesLoader } }) => {
         if (sale_ids && sale_ids.length > 0 && acquireable) {
           return salesLoader(id, {
             id: sale_ids,
             is_auction: true,
-            auction_state: "open",
+            live: true,
+          }).then(sales => {
+            return sales.length > 0
```

Comment:
We actually don't need to make any additional calls/do any logic since we filter out artworks that are not `acquireable` above. All we care about in this case is that there is a sale that meets the above criteria.